### ISSUE TITLE
Misses mobilize.us tracker

### ISSUE THREAD

**Issue by j-f1** [apparition47/MailTrackerBlocker#79]
The alert says “MailTrackerBlocker did not detect any trackers in this email.” but the HTML source shows this.

```html
<img width="1px" height="1px" alt="" src="https://email.newsletter.mobilize.us/o/eJwVjEsOgyAUAE8jSwI8sLpgYWx7DcPnEWlQDGJse_razSQzi_FaceNUIFELJjgD3v3JBOUUnt0Ij7EfYBBK3GUj2YrnnrBWLHTJNqb4RXrsZNYtOBCOBeNtf5NWMI7OMC6kahUABFL0C3300cxTyO9rZUs-V4r-IFWfuSTfwLCV7C_FxcQ01c-GV5P9DxElMgM">
```

**Comment by apparition47** [comment]
Odd. This one should definitely be picked up by the generic filter rule. Not able to reproduce either. Are you on the latest `0.3.19` version? If so, could I trouble you for the `eml` export? Thanks again for reporting.

**Comment by apparition47** [comment]
Fixed the generic 1x1 rule in 0.3.22. Will need to look into mobilize a bit more to add it to the list.
